src/index.js

```diff
@@ -0,0 +1,96 @@
+const addTaskButton = document.getElementById('add-task-btn');
+const newTaskInput = document.getElementById('new-task-input');
+const todoList = document.getElementById('todo-list');
+const clearCompletedButton = document.getElementById('clear-completed-btn');
+
+function deleteTask(todoItem) { todoItem.remove(); }
+
```

Comment:
- [ ] Please kindly check to ensure that your list should appear in order of the index values for each task, because at the moment it does not appear in order.